CARL/source/DynamicTimeWarping.h

```diff
@@ -91,10 +95,11 @@ namespace carl::DynamicTimeWarping
         size_t minimumIdx = disallowedImageLength;
         for (size_t idx = disallowedImageLength; idx < currentRow.size(); ++idx)
         {
-            if (currentRow[idx] < minimum)
+            size_t connectionsCount = std::max(idx, b.size());
+            if (currentRow[idx] / connectionsCount < minimum)
             {
                 minimumIdx = idx;
-                minimum = currentRow[minimumIdx];
+                minimum = currentRow[minimumIdx] / connectionsCount;
```

Comment:
The bug fixed by this --- that injective distance, with its inherent ability to accept matches with varying numbers of pairings, will overwhelmingly bias toward short projections unless normalization is considered --- had been affecting CARL since injective DTW was introduced at the very beginning.